Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidSensors.cs

```diff
@@ -276,6 +279,48 @@ public class AndroidGameRotationVector : AttitudeSensor
     public class AndroidStepCounter : StepCounter
     {
     }
+
+    /// <summary>
+    /// Hinge angle sensor device on Android.
+    /// This sensor is usually available on foldable devices.
+    /// </summary>
+    /// <seealso href="https://developer.android.com/reference/android/hardware/Sensor#TYPE_HINGE_ANGLE/>
+    [InputControlLayout(stateType = typeof(AndroidSensorState), variants = nameof(AndroidSensorType.HingeAngle), hideInUI = true)]
+    public class AndroidHingeAngle : Sensor
+    {
+        /// <summary>
+        /// The angle in degrees on how much the phone is unfolded
```

Comment:
Would recommend that this explicitly states that the unit to be expected is degrees?

Reply:
I don't understand the question, comment already mentions degrees ? Can you rephrase or elaborate?

Reply:
Sorry for that, seems like my eyes or brain completely malfunctioned